faheem/infra/lambda/constants.py

```diff
@@ -0,0 +1 @@
+URLS_TO_MONITOR = ['https://www.skipq.org/', 'https://www.google.com/']
```

Comment:
As our goal is just to test availability and latency of skipq.org, checking google.com will use more resources and not required as well.